source/persistence/MessageController.cpp

```diff
@@ -97,7 +97,7 @@ void MessageController::setTable ( const QString &table_name )
 }
 
 // FIXME use direction enum
-bool MessageController::addMessage(const QString &id, const QString &jid, const QString &resource, const QString &message, const QString &type, unsigned int direction, qint64 timestamp)
+bool MessageController::addMessage(const QString &id, const QString &jid, const QString &resource, const QString &message, const QString &type, bool isGroupMessage, unsigned int direction, qint64 timestamp)
```

Comment:
try to get rid of the additional attribute in the database. See the comment at main.qml